scripts/release

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+PUBLIC_BUCKET="trussworks-tools-us-west-2"
+GITHUB_USER="trussworks"
+GITHUB_REPO="terraform-aws-iam-sleuth"
+ZIP_NAME="deployment.zip"
+NEW_TAG="1.0.7"
+
+ghr -u $GITHUB_USER -r $GITHUB_REPO $NEW_TAG $ZIP_NAME
+echo "Created a new release with ${NEW_TAG}"
+
+if [[ -n "${CIRCLECI+x}" ]]; then
+    // assume roles
+    temp_role=$(aws sts assume-role \
+                      --role-arn arn:aws:iam::313564602749:role/circleci \
+                      --role-session-name circleci )
+              export AWS_ACCESS_KEY_ID=$(echo $temp_role | jq .Credentials.AccessKeyId | xargs)
+              export AWS_SECRET_ACCESS_KEY=$(echo $temp_role | jq .Credentials.SecretAccessKey | xargs)
+              export AWS_SESSION_TOKEN=$(echo $temp_role | jq .Credentials.SessionToken | xargs)
+    aws s3 cp $ZIP_NAME s3://$PUBLIC_BUCKET/iam-sleuth/$NEW_TAG/$ZIP_NAME --sse AES256
+else
+    AWS_VAULT_KEYCHAIN_NAME=login aws-vault exec trussworks-prod -- aws s3 cp $ZIP_NAME s3://$PUBLIC_BUCKET/iam-sleuth/$NEW_TAG/$ZIP_NAME --sse AES256
+fi
+
+echo "Uploaded ${ZIP_NAME} to S3 bucket ${PUBLIC_BUCKET}"
```

Comment:
I suggest outputting the full s3 path to the file... additionally maybe verify the output of the uploads in case there was. a problem in upload.

Reply:
Just gonna remove this since the command outputs a similar message anyways.